io.fairyproject.platforms/core-platform/build.gradle

```diff
@@ -31,6 +31,7 @@ dependencies {
     compileOnly "org.ow2.asm:asm:" + findProperty("asm.version")
     compileOnly "org.ow2.asm:asm-commons:" + findProperty("asm.version")
 
+    implementation "commons-io:commons-io:2.7"
```

Comment:
I think it can be made as runtime library download instead of shade directly?

Reply:
I added it to LibraryHandler, Library.
```java
// APACHE
this.downloadLibraries(true,
        Library.COMMONS_IO
);
```
```java
COMMONS_IO = new Library(
            "commons-io",
            "commons-io",
            "2.7",
            null
    );
```

Reply:
yep looks nice!